### PR TITLE
VR-5525 Endpoint.__repr__()

### PR DESCRIPTION
**Pull request by nhatsmrt**

### CODE REVIEW

File: client/verta/tests/test_endpoint/test_endpoint.py [hunk 1]
```diff
@@ -68,6 +67,21 @@ def test_get_status(self, client):
...
 
+    def test_repr(self, client, experiment_run, model_for_deployment):
+        experiment_run.log_model(model_for_deployment['model'], custom_modules=[])
+        experiment_run.log_requirements(['scikit-learn'])
+
+        path = verta._internal_utils._utils.generate_default_name()
+        endpoint = client.set_endpoint(path)
+        endpoint.update(experiment_run, DirectUpdateStrategy())
+
+        str_repr = str(endpoint)
```
Comment (convoliution) [inline client/verta/tests/test_endpoint/test_endpoint.py]: Pretty sure this makes no difference at all, but at least for consistency:
```suggestion
        str_repr = repr(endpoint)
```

File: client/verta/verta/_deployment/endpoint.py [hunk 2]
```diff
@@ -15,8 +16,11 @@ def __init__(self, conn, conf, workspace, id):
         self.id = id
 
     def __repr__(self):
-        # TODO: print full info
-        return "<Endpoint \"{}\">".format(self.path)
+        return '\n'.join((
+            "path: {}".format(self.path),
+            "id: {}".format(self.id),
+            "stage: {}".format(json.dumps(self.get_status(), indent=4)),
```
Comment (convoliution) [inline client/verta/verta/_deployment/endpoint.py]: I know I left this as an exercise for you 😆
The stage is kinda something we're trying to abstract away from the user for now, and also printing out the full status here (in JSON, too) may be a little overwhelming.

I'd bring `get_status()['status']`, `'date_created'`, `'date_updated'` up to the top level, and only show `'components'` here.

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/verta/verta/_deployment/endpoint.py (hunk 2 above)
Comment (nhatsmrt) [reply client/verta/verta/_deployment/endpoint.py]: Done-zo!

File: client/verta/tests/test_endpoint/test_endpoint.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/tests/test_endpoint/test_endpoint.py]: fixed.

**Review by convoliution — APPROVED** [review]

**Comment by nhatsmrt** [comment]
Conflict resolved.

**Pull request merged**